tools/parse-release-notes

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const { GITKEEP_FILE, UNRELEASED_DIR, RELEASE_NOTES_DIR } = require('./constants')
+const { program } = require('commander')
+const yaml = require('js-yaml')
+
+const DEFAULT_LANGUAGE = 'en'
```

Comment:
Should be 'de' like in the translations of the app.

Reply:
Are you sure? Store descriptions are nearly everywhere english if the language is not provided.